scanner/config/config.go

```diff
@@ -157,6 +159,13 @@ func (c *MatcherConfig) validate() error {
 			return fmt.Errorf("indexer_addr: failed to parse address: %w", err)
 		}
 	}
+	if c.DefinitionsURL == "" {
```

Comment:
not important for this PR, just in the future we may not allow this to be empty. I started work related to this and made very similar changes https://github.com/stackrox/stackrox/pull/9417/files

whether we change it later or not doesn't matter for this PR. just giving some extra info

Reply:
Thinking about it, is this change necessary for this PR?

Reply:
No need here, but we must do it to release Scanner V4. I left a related TODO in this PR. Either here or there can go in first.